remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/gesture/ShakeListener.java

```diff
@@ -0,0 +1,52 @@
+package com.google.android.libraries.remixer.ui.gesture;
+
+import android.content.Context;
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+import android.support.v4.app.FragmentActivity;
+
+import com.google.android.libraries.remixer.ui.view.RemixerFragment;
+
+public class ShakeListener implements SensorEventListener {
+    private double lastMagnitude;
+    // TODO(nicksahler): Remove `lastSpike` once `show` bug is fixed
```

Comment:
Add the bug number here, or a link. Sorry i wasn't clear on that. this gives better context.